docs/getting_started.rst

```diff
@@ -1,2 +1,136 @@
+
+::
+
+    mkdir sdk-demo && cd sdk-demo
+    npm init
+
+Follow ``npm init`` wizard to initialize your project. Then install the SDK, which is the only dependency we need to build a simple application.
+
+::
+
+    npm add @polymathnetwork/sdk@2.0.1-beta.24
+
+Finally, add an entry js file such as ``index.js``, which imports the SDK API as well as some browser utils that we're going to use later.
+
+::
+
+    import { Polymath, browserUtils } from '@polymathnetwork/sdk';
+
+Instantiate and initialize Polymath client
+------------------------------------------
+
+Before we use Polymath SDK, we need to `connect` to the deployed smart contracts on the desired Ethereum network. 
+
+::
+
+    const privateKey = process.env.PRIVATE_KEY;
+    const networkId = 1; // Or browserUtils.getNetworkId()
+    const networkConfigs = {
+        1: {
+            polymathRegistryAddress: '0xdfabf3e4793cd30affb47ab6fa4cf4eef26bbc27',
+            httpProviderUrl: 'https://mainnet.infura.io/v3/[INFURA_PRODUCT_ID]',
+            privateKey
+        },
+        42: {
+            polymathRegistryAddress: '0x5b215a7d39ee305ad28da29bf2f0425c6c2a00b3',
+            httpProviderUrl: 'https://kovan.infura.io/v3/[INFURA_PRODUCT_ID]',
+            privateKey
+        },
+    };
+    const config = networkConfigs[networkId];
+    const polyClient = new Polymath();
+    await polyClient.connect(config);
+
+
+``PolyClient.connect()`` accepts three parameters:
+
+- ``PolymathRegistryAddress``, which is the address of the main registry that keeps track of core components of the system, including the addresses of other registries. 
+- ``httpProviderUrl``, which is an Ethereum network client that enables you to view and interact with the blockchain, via HTTP. That node can be a local ``geth`` or ``parity`` network client, or a 3rd-party provider such a ``Infura``.
+- ``privateKey`` this is the private key used to sign transactions sent from the SDK. 
+
+**On browsers**, some plugins such as Metamask, automatically injects a Web3 instance for your webpage to use. In that case, you can omit passing `httpProviderUrl` and `privateKey`. The SDK will use the injected provider behind the scenes.
+
+Additionally, you can detect the currently selected network using ``browserUtils.getNetworkId()``, which returns network's ID. e.g `1` for mainnet.
+
+Deploying your first security token
+-----------------------------------
+
+There's a multitude of modules and features provided by Polymath smart contracts. Many of them are abstracted by the SDK for simpler usage. In this section, we are going to interact with ``SecurityTokenRegistry`` contract to create our first token.
+
+::
+
+    try {
+        const reservation = await polyClient.reserveSecurityToken({
+            symbol: 'ABC123',
+            name: 'ABC 123 Inc.',
+        });
+        await reservation.run(); // Will run sequentially every transaction required
+
+        const creation = await polyClient.createSecurityToken({
```

Comment:
This function is no longer exposed directly from the polyClient, nor does the reservation require a name

The correct way of doing this would be 

```
const reservationQ = await polyClient.reserveSecurityToken({
      symbol: 'ABC123',
});
const reservation = await reservation.run();
const creationQ = await reservation.createSecurityToken({
      name: 'ABC 123 Inc.',
      detailsUrl: 'http://example.com',
      divisible: true,
});
const token = await creationQ.run();
```

Reply:
Done